src/backlight/strategies/filter.py

```diff
@@ -30,3 +31,29 @@ def limit_max_amount(trades: Trades, max_amount: int) -> Trades:
         current_amount = next_amount
 
     return trades[~trades["_id"].isin(deleted_ids)]
+
+
+def skip_entry_by_spread(
+    trades: Trades, mkt: AskBidMarketData, max_spread: float
+) -> Trades:
+    """Skip entry by spread.
+
+    Args:
+        trades: Trades
+        mkt: Market data for ask/bid prices
+        max_spread: More than the value, skip entry
+    Result:
+        Trades
+    """
+    assert max_spread >= 0.0
+    assert trades.index.unique().isin(mkt.index).all()
+
+    spread = mkt.spread
+
+    deleted_ids = []  # type: List[int]
+    for i in trades.ids:
+        entry = trades.get_trade(i).index[0]
+        if spread.loc[entry] > max_spread:
```

Comment:
What about using `spread.at` ? It's a bit faster and it will not change the results in my understanding.

I didn't found other points to improve.